vlingo-reactive-messaging-patterns/src/main/java/io/vlingo/reactive/messaging/patterns/messagebus/NotificationInterest.java

```diff
@@ -0,0 +1,97 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+package io.vlingo.reactive.messaging.patterns.messagebus;
+
+/**
+ * NotificationInterest
+ *
+ * @author brsg.io
+ * @since Oct 31, 2018
+ */
+public class NotificationInterest
```

Comment:
These JavaBean classes seem to be quite limited in value. Note that in the Scala code these types are messages being sent between actors. In this code it seems to only convey some identity, and I am not sure if they are needed for that. (Not sure they aren't needed, but the naming needs to convey more meaning if they are needed.)